vignettes/articles/minimalgut.Rmd

```diff
@@ -194,7 +193,7 @@ tse[["sliding_divergence"]] <- colData(tse) |>
 After calculating the moving average of divergences, we can visualize the
 result in a similar way to our previous approach.
 
-```{r}
+```{r, message = FALSE, warning = FALSE}
```

Comment:
This comes in every chunk, could we set this globally in the beginning of the Quarto doc?

Reply:
yes. can be added to `opts_chunk$set` for global effect